### DISCUSSION TITLE
Is there a way to convert an animated GIF file to a series of static GIF files?

### DISCUSSION THREAD

**Discussion by lkintact** [ImageMagick/ImageMagick#7653] [Help] [answered]
### ImageMagick version

7.1.1-38 Q16-HDRI x64 b0ab922:20240901

### Operating system, version and so on

Windows 10 Home

### Description

Hi everyone,

I would like to convert a GIF animation to a series of static GIF files.
I realize I can do something like `magick.exe animation.gif static.png` and then convert the PNG files into GIFs, but is there a way to directly convert from GIF to GIF?

**Comment by fmw42** [comment, marked as answer]
`magick.exe animation.gif -coalesce +adjoin image.gif`

The result will be image-N.gif where N=0,1...N images.

**Comment by lkintact** [comment]
Thank you, @fmw42, that worked!
